src/test/java/org/jenkinsci/plugins/workflow/graphanalysis/ChunkFinderWithoutChunks.java

```diff
@@ -0,0 +1,26 @@
+package org.jenkinsci.plugins.workflow.graphanalysis;
+
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+/**
+ * For test use: a ChunkFinder that never returns chunks, to use in testing parallel handling only.
+ */
+public class ChunkFinderWithoutChunks implements ChunkFinder {
```

Comment:
@jglick Maybe not a play, but the ChunkFinderWithout chunks follows up on Being and Nothingness, exploring the metaphysical ramifications of finding Chunks without Finding them. 

Or maybe I just couldn't come up with a clever name for it.  Ideas?  Maybe NoOpChunkFinder?